com.xored.f4.fantom-feature/feature.xml

```diff
@@ -1,20 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <feature
       id="com.xored.f4.fantom"
-      label="Embedded Fantom"
+      label="Fantom Interpreter"
```

Comment:
Hmm... nothing showed up just now when I did a File Search. But yeah, probably best if you double check!